packages/module/src/RemoveModal/RemoveModal.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { Button, Checkbox, Modal, ModalProps, ModalVariant, Split, SplitItem, Stack, TextContent } from '@patternfly/react-core';
+import { createUseStyles } from 'react-jss'
+
+
+const useStyles = createUseStyles({
+  rbacDeleteIcon: {
+    marginRight: 'var(--pf-v5-global--spacer--xs)'
+  },
+})
+
+export interface RemoveModalProps extends Omit<ModalProps, 'ref'|'children'> {
+  /** Title for the Remove Modal. */
+  title: string;
+  /** Text inside the Remove Modal */
+  text: string;
+  /** Custom label for the cancel action button */
+  confirmButtonLabel: string;
```

Comment:
can you please add also the cancelButtonLabel?

```suggestion
/** Custom label for the confirm action button */
  confirmButtonLabel? : string;
  /** Custom label for the cancel action button */
  cancelButtonLabel? : string;
```